unicornia/src/Blocks/components/menu/styles/horizontal/menu-horizontal-style.scss

```
@@ -28,4 +28,10 @@ $menu-horizontal: (
 
 		@include link($menu-horizontal, link-modifiers);
 	}
+
+	&__item--active {
```

```suggestion
	&__item {
		&--active {
```

Would be the correct way of adding modifiers 🙂